docs/docs/aztec/concepts/storage/partial_notes.md

```diff
@@ -47,11 +82,9 @@ $$
 P_a' := \text{funded amount}*G_{amount} + \text{alice address}*G_{address} + \text{rand}_a*G_{randomness}
 $$
 
-Where did that $\text{rand}_a$ come from? Well from Alice of course. So we can't trust it is random. So we hash it with Alice's address and emit it as a nullifier.
```

Comment:
Mitch originally though this might be a problem because at the time we did not have nonces in publicly injected note hashes and for this reason if user would use the same randomness twice and the note would end up having the same amount you would inject 2 equal notes into a note hash tree and effectively lose one of those because you could spend only 1 of those (because nullifier would be the same).

A while ago Leila implemented nonces in the public note hashes so this is no longer a problem.